src/core/server/logging/logging_config.test.ts

```diff
@@ -9,7 +9,35 @@
 import { LoggingConfig, config } from './logging_config';
 
 test('`schema` creates correct schema with defaults.', () => {
-  expect(config.schema.validate({})).toMatchSnapshot();
+  expect(config.schema.validate({})).toMatchInlineSnapshot(
+    { json: expect.any(Boolean) }, // default value depends on TTY
```

Comment:
Had to use a matcher for `json` as the legacy logging schema is defining

```ts
json: schema.boolean({
    defaultValue: !process.stdout.isTTY,
}),
```

which caused different snapshots locally and on CI... I assumed this was acceptable instead of mocking `process.stdout`.